contracts/Controller.sol

```diff
@@ -292,7 +302,11 @@ contract Controller is ReentrancyGuard, Ownable {
      * @notice deposit long option into vault
      * @param _args DepositArgs structure
      */
-    function _depositLong(Actions.DepositArgs memory _args) internal returns (MarginAccount.Vault memory) {
+    function _depositLong(Actions.DepositArgs memory _args)
+        internal
+        isValidVaultId(_args.owner, _args.vaultId)
```

Comment:
Can you make this a require and not a modifier? Little harder to read

Reply:
u mean the `isValidVaultId` ? I think it will be used in many functions, better to keep it a modifier ?!

Reply:
yeah, well its a little confusing to read that + is authorized and all as modifiers. Its easier to read them all if they are in the function.

Reply:
When there are multiple checks, it becomes confusing to keep track of what has been done and not if we use too many modifiers.

Reply:
it can still be a function, but doing it as a modifier makes it confusing

Reply:
@antoncoding thoughts ?

Reply:
I think internal function is easier to read, I feel like people usually use modifier for access control (to see who can call this function or not). I put other checks in internal functions

Reply:
updated this.